src/library/Uno.Material/Styles/Application/Colors.xaml

```diff
@@ -66,18 +68,38 @@
 					 Color="{ThemeResource SurfaceColor}"/>
 	<SolidColorBrush x:Key="ErrorBrush"
 					 Color="{ThemeResource ErrorColor}"/>
-
+	<SolidColorBrush x:Key="OnErrorBrush"
+					 Color="{ThemeResource OnErrorColor}"/>
+	<SolidColorBrush x:Key="OverlayBrush"
+					 Color="{ThemeResource OverlayColor}"/>
+
+	<!-- Primary Brushes -->
```

Comment:
You can remove the comment below
	<!-- OnSurfacePrimary Brushes -->
	<!-- TODO : Understand them and add -->		<!-- TODO : Understand them and add -->
	<!-- They are brushes following the same pattern as OnSurface		<!-- They are brushes following the same pattern as OnSurface
		but instead of being based as a gray shade, they would be a shade of the primary color?			but instead of being based as a gray shade, they would be a shade of the primary color?
	-->		-->